test/integration/Templates.test.ts

```diff
@@ -90,4 +92,25 @@ describe('Client - Templates', function () {
             expect(result.Message).to.eq('OK');
         });
     });
+
+    describe('push templates', () => {
+        it("non existing servers error", () => {
```

Comment:
Inconsistent quotes ([tslint](https://palantir.github.io/tslint/) would help)

Reply:
grazias, ran tslint to fix